src/components/Button/Button.tsx

```diff
@@ -7,8 +11,10 @@ interface IButtonProps {
 }
 
 const Button: FC<IButtonProps> = ({ className, disabled, children}) => {
+  const button = cn('Button')
+
   return (
-    <button className={className}>{children}</button>
+    <button className={cx(button(), className)}>{children}</button>
```

Comment:
Ты написал рабочий код, ок, но лучше использовать меньше библиотек. Я сам не так давно узнал об этом способе. Можно склеить классы библиотекой `@bem-react/classname`
```suggestion
    <button className={button(null, [className])}>{children}</button>
```
<br>

Пример такого [в @yandex-ui](https://github.com/bem/yandex-ui/blob/d0ac21393d6defbff5e6dcbd499de2ac7f4d5cde/src/Link/Link.tsx#L123)